### ISSUE TITLE
508-defect-2 [SCREENREADER, MOBILE DESIGN] Page MUST contain a level-one heading

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#11888]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

For smaller viewports, page **must** contain a level-one heading. The H1 that appears in the desktop view is removed for viewport widths narrower than 768 pixels wide. Screen reader users sometimes navigate page structure by headings, and having an h1 on all page is a standard best practice.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to confidently navigate the page hierarchy by headings

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov in browser
2. Log in as user36
3. Navigate to Profile
4. Resize the browser to below 768 pixels wie
5. Run the axe browser scan
6. Verify the error appears, "Page must contain a level-one heading"

## Possible Fixes (optional)

Within the mobile nav code, replace the `<strong>Your profile menu</strong>` with `<h1>Your profile</h1> menu` and style as necessary.

![Screen Shot 2020-07-31 at 11 13 40 AM](https://user-images.githubusercontent.com/57469/89050254-7bf09500-d320-11ea-9b9d-f08f508d1ee8.png)

## WCAG or Vendor Guidance (optional)

* [Deque University - Page must contain a level-one heading, WCAG: Best Practice](https://dequeuniversity.com/rules/axe/3.5/page-has-heading-one?application=AxeChrome)

**Comment by mshea0606** [comment]
@tressaellen and @erik to meet with Jennifer

**Comment by erikphansen** [comment]
@jenstrickland this should be resolved now.

**Comment by erikphansen** [comment]
@jenstrickland should validate this.

**Comment by jenstrickland** [comment]
This issue is resolved. There is a typo in the code where the `div data-widget-type="banners" data-homepage-banner-content="` ends without closing before the `<p>`. Do you want me to open a ticket to address, or just handle it?
![Screen Shot 2020-08-14 at 3 57 33 PM](https://user-images.githubusercontent.com/57469/90287991-f48a4200-de46-11ea-9f02-b8b4c7ac89d6.png)

**Comment by erikphansen** [comment]
I think that might be the public website team's code?

**Comment by jenstrickland** [comment]
You're right, as usual! Thank you!
